packages/node_modules/@webex/plugin-meetings/src/common/collection.js

```diff
@@ -0,0 +1,90 @@
+import {MEETINGS} from '../constants';
+
+/**
+ * @class Collection
+ */
+export default class Collection {
+  namespace = MEETINGS;
```

Comment:
I don't think you need to namespace the class unless it extends one of the plugins.

Reply:
okay, we are doing it all over the place right now, so ill keep it and we can re-evaluate all of them at a later date.